docs/calls.md

```diff
@@ -50,9 +54,9 @@ EXAMPLES
 
 _See code: [src/commands/calls/index.ts](https://github.com/APImetrics/APIm-CLI/blob/v0.2.1/src/commands/calls/index.ts)_
 
-## `apimetrics calls create`
+### `apimetrics calls create`
 
-Create a new API call.
+Create a new API call in the Project. **Project IDs** can be found in the Project Settings under the Admin section or by using the command `apimetrics projects -x --no-truncate`.
```

Comment:
I think these would probably be better in the flag description not in the command description